src/services/tray.ts

```diff
@@ -46,13 +47,14 @@ const registerVisibilityListener = () => {
         isVisible = false;
     });
 
-    UPDATE_TRAY_EVENTS.forEach(eventName => {
-        mainWindow.on(eventName, updateTrayMenu)
+    UPDATE_TRAY_EVENTS.forEach((eventName) => {
+        mainWindow.on(eventName as any, updateTrayMenu)
```

Comment:
We should strive to not have any in the codebase, I realize that this is specific case is due to the electron types being declared a bit wacky, but as here is the only place were multiple event listeners are registered,  declaring them by hand instead of programmatically would make the code a bit uglier, but would fix the need for an any
```
mainWindow.on('minimize', updateTrayMenu)
// etc etc
```

Reply:
Good point. I think it makes the code actually slightly more readable, since the original implementation was a bit of over-engineering.